functions/api/v1/statuses.ts

```diff
@@ -128,5 +128,5 @@ export async function handleRequest(
 		...cors(),
 		'content-type': 'application/json; charset=utf-8',
 	}
-	return new Response(JSON.stringify(res), { headers })
+	return new Response(JSON.stringify({ ...res, mastodonId: note.mastodonId }), { headers })
```

Comment:
mastodonId is id already

Reply:
I don't think so, I saw that it is different from the id we provide in the dummy data statuses and at each db setup it changes 😕

Reply:
for example if I print the map in which I map the ids we provide in dummyData to mastodonIds I get this sort of result:
![Screenshot 2023-01-18 at 14 22 11](https://user-images.githubusercontent.com/61631103/213196080-04bf6623-51cd-4654-9f6d-02fc9393facf.png)

![Screenshot 2023-01-18 at 14 23 06](https://user-images.githubusercontent.com/61631103/213196095-8df46330-aaf3-4519-b0e4-a3c46cb22672.png)

![Screenshot 2023-01-18 at 14 23 31](https://user-images.githubusercontent.com/61631103/213196140-17b3eb5f-ad5d-4120-833e-ca05c937da03.png)

as you can see our id is always the same but the mastodonId seems to change every time

Reply:
anyways it doesn't matter as I am going to refactor the ui tests to use the lower level api there